### PR TITLE
New package: WorkFlowy 4.0.2302081114.991

### PR DESCRIPTION
**Pull request by stkb**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97056)

### CODE REVIEW

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114.991
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.installer.yaml:5]:
```suggestion
PackageVersion: 4.0.2302081114
```
Must match the version shown in Control Panel / Apps&Features

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114.991
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.0.2302081114
```
Must match the version shown in Control Panel / Apps&Features

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114.991
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.yaml:5]:
```suggestion
PackageVersion: 4.0.2302081114
```
Must match the version shown in Control Panel / Apps&Features

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder name to match the new PackageVersion. The folder under `manifests/w/WorkFlowy/WorkFlowy` should be renamed from `4.0.2302081114.991` -> `4.0.2302081114`.

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by stkb** [comment]
@mdanish-kh Thanks and yeah, I wasn't sure about the version number but had stuck with the one wingetcreate had picked up.

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114
+PackageLocale: en-US
+Publisher: WorkFlowy
+PackageName: WorkFlowy
+License: Proprietary
+Copyright: Copyright © 2023 WorkFlowy
+ShortDescription: Minimilist-design inifinite bullet-list outliner
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Workflowy offers a simpler way to stay organized. If you have a crazy job or an ambitious project, we will be your trusty sidekick.
```
There are typos in the ShortDescription. I would suggest sticking to this one from the official website.

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114
+PackageLocale: en-US
+Publisher: WorkFlowy
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml:7]:
```suggestion
Publisher: WorkFlowy
PublisherUrl: https://workflowy.com/
PrivacyUrl: https://workflowy.com/privacy-policy
```
Metadata

File: manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+PackageName: WorkFlowy
```
Comment (mdanish-kh) [inline manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml:8]:
```suggestion
PackageName: WorkFlowy
PackageUrl: https://workflowy.com/
```
Metadata

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Since I discovered the typo, I'll take this moment to suggest some useful metadata. (apologies for re-requesting changes as I couldn't review thoroughly earlier)

**Comment by mdanish-kh** [comment]
> @mdanish-kh Thanks and yeah, I wasn't sure about the version number but had stuck with the one wingetcreate had picked up.

WinGet requires the version to match Control Panel / Apps&Features. Since wingetcreate doesn't have the ability to know in advance what would be that value, it picks up the file version from the installer. Many times publishers set it to a different value than what they write to the "DisplayVersion" key of the Registry.

**Comment by stkb** [comment]
@mdanish-kh Thanks for all the feedback and work you've put in. I'll know better next time :)

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**